src/search-modal/SearchUI.test.tsx

```diff
@@ -487,4 +487,39 @@ describe('<SearchUI />', () => {
       expect(queryByLabelText(/^Lightcast/i)).toBeInTheDocument();
     });
   });
+
+  describe('searchKeywords', () => {
+    let rendered: RenderResult;
+    beforeEach(async () => {
+      rendered = render(<Wrap><SearchUI {...defaults} /></Wrap>);
+      // Wait for initial search request
+      await waitFor(() => { expect(fetchMock).toHaveFetchedTimes(1, searchEndpoint, 'post'); });
+    });
+
+    it('should update search results when keywords are entered', async () => {
+      const { getByRole, getByText } = rendered;
```

Comment:
@DanielVZ96  It's [recommended to use ](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen)`screen` here